policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml

```diff
@@ -1,6 +1,7 @@
 description: The tax rate on Part A short-term capital gains.
 values:
   2002-01-01: 0.12
+  2023-01-01: 0.085
 metadata:
   label: MA short-term capital gains rate
   reference:
```

Comment:
can you change the below one to `(a)(1)` and leave a comment indicating that it does not yet reflect the 8.5% change as of 2023-12-22?